### DISCUSSION TITLE
Markdown details tag with default summary has incorrect cursor

### DISCUSSION THREAD

**Discussion by jstrieb** [community/community#23983] [New to GitHub]
<p>This is a (minor) bug report.</p>
<p>On <a href="https://github.com" rel="noopener nofollow ugc">github.com</a>, with the default GitHub stylesheet, the <code>&lt;details&gt;</code> tag only has <code>cursor: pointer</code> if a <code>&lt;summary&gt;</code> tag is included explicitly. If there is no <code>&lt;summary&gt;</code> tag, the cursor does not indicate that the dropdown is clickable. Here is an example of the incorrect style:</p>
<details>
<p>This is a test</p>
</details>
<p>The code for this incorrect version is:</p>
<pre><code class="lang-markdown">&lt;details&gt;

This is a test

&lt;/details&gt;
</code></pre>
<p>Compare that to explicitly including a <code>&lt;summary&gt;</code> tag, which behaves the way it is supposed to:</p>
<details>
<summary>Details</summary>
<p>This is another test</p>
</details>
<p>The code for the second example is:</p>
<pre><code class="lang-markdown">&lt;details&gt;

&lt;summary&gt;Details&lt;/summary&gt;

This is another test

&lt;/details&gt;
</code></pre>
<p>This can be fixed by adding the following CSS to the stylesheet:</p>
<pre><code class="lang-css">details {
  cursor: pointer;
}
</code></pre>

**Comment by ernest-phillips** [comment]
<p>Thanks for letting us know about this <a class="mention" href="/jstrieb">@jstrieb</a> !</p>
<p>Could you supply a link to where you see this happening?<br>
A screenshot would be most helpful.</p>
<p>Thanks!</p>

**Comment by jstrieb** [comment]
<p>Hi <a class="mention" href="/ernest-phillips">@ernest-phillips</a>, thanks for the response!</p>
<p>In my browser (Firefox 85.0.2 (64-bit)), it’s actually happening in the post above – I think the <a href="https://github.community">github.community</a> stylesheet has the same issue. I can still provide screenshots of it happening on <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> though, and can also provide a Gist demo if that is preferable.</p>
<p>Thanks again!</p>

**Comment by ernest-phillips** [comment]
<p>Thanks for that additional context . The Community site is run on Discourse. I will look into seeing if we can make a change there.</p>
<p>The screenshot of this behavior on <a href="http://GitHub.com">GitHub.com</a> would be most helpful in relaying the info to our engineers.</p>

**Comment by jstrieb** [comment]
<p>Incorrect behavior screenshot:<br>
<img src="https://user-images.githubusercontent.com/7355528/181080497-d03eda2a-a617-47d5-ba83-eada2d88a6e3.png" alt="image" data-base62-sha1="2BqKorEevO68Javj0la1TbEPJKj" width="354" height="290"></p>
<p>Comparison to explicit inclusion of <code>&lt;summary&gt;</code>:<br>
<img src="https://user-images.githubusercontent.com/7355528/181080498-fcbd8bdd-4eac-40b7-8800-c44645feec86.png" alt="image" data-base62-sha1="9bcjhLNhOYhXyRe8UdR16trJpBy" width="354" height="290"></p>
<p>Screenshots taken from this demonstrative gist:<br>
<a href="https://gist.github.com/jstrieb/ccb5c73ff92be6bc066145e80f517313" rel="noopener nofollow ugc">https://gist.github.com/jstrieb/ccb5c73ff92be6bc066145e80f517313</a></p>

**Comment by ernest-phillips** [comment, marked as answer]
<p><a class="mention" href="/jstrieb">@jstrieb</a>  thanks for the thorough breakdown of this issue.<br>
I have reported this to our engineering team.</p>
<p>I can’t say when this will be fixed, and when it does it will likely not have any mention in a changelog. Please be assured that this information has been delivered to the proper team.</p>
